packages/cozy-ccc-libs/package.json

```diff
@@ -18,12 +18,14 @@
   "homepage": "https://github.com/konnectors/libs#readme",
   "scripts": {
     "lint": "eslint 'src/**/*.js'",
-    "test": "jest"
+    "build": "babel src/ -d dist/ --copy-files --verbose --ignore '**/*.spec.js','**/*.spec.jsx'",
+    "test": "jest src"
```

Comment:
I think adding this `src` will break stuff like: `yarn test --watch` for instance. No?

Reply:
I don't see why it would break yarn test --watch since yarn jest src --watch would still work. But now I use `testPathIgnorePatterns` in jest config to ignore `dist` path which is probably better anyway

Reply:
but you still specify the `src` folder. 

So if I write `yarn test myFile.js --watch` it'll not work. And I don't want to use `yarn jest myFile.js --watch` because maybe you have a specific config and we need to use it.